sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -1062,6 +1063,23 @@ public CosmosClientBuilder clientTelemetryConfig(CosmosClientTelemetryConfig tel
         return this;
     }
 
+    /**
+     * Sets a custom serializer that should be used for conversion between POJOs and Json payload stored in the
+     * Cosmos DB service. The custom serializer can also be specified in request options. If defined here and
+     * in request options the serializer defined in request options will be used.
+     * @param serializer the custom serialzier to be used for payload transformations
+     * @return current CosmosClientBuilder
+     */
+    public CosmosClientBuilder setCustomSerializer(CosmosItemSerializer serializer) {
```

Comment:
nit: Do we follow the fluent pattern here?

Reply:
Fixed